example/gradle.properties

```diff
@@ -1,21 +1,20 @@
-## ## keep in sync with gradle.properties in root ## ##
```

Comment:
I suspect there is still plenty of stuff to keep in sync?

Reply:
Added the comment back